src/sass/components/_verification_level.scss

```diff
@@ -1,56 +1,53 @@
-/*
- * Verification level.
- */
-
+// Verification level
```

Comment:
Here again the comment is a bit wrong. The removed comment applies to the entire file, while after removing the empty line it looks like it only applies to the .verification-dots class. @lgrahl do you know of a way to create file-level comments?

There are other cases of this issue in the diff.